Core/Object Arts/Dolphin/MVP/ResourceIdentifierTest.cls

```diff
@@ -10,6 +10,18 @@ ResourceIdentifierTest comment: ''!
 !ResourceIdentifierTest categoriesForClass!Unclassified! !
 !ResourceIdentifierTest methodsFor!
 
+createResourceNamed: aString for: aView
+	| filer rid resource |
+
+	filer := STLOutFiler on: Array writeStream.
+	filer nextPut: aView.
+	aView destroy.
+	resource := filer stream contents.
+	rid := ResourceIdentifier class: Presenter name: aString.
```

Comment:
Unfortunately this is going to leave the Presenter class and base package marked as dirty when people run the test. You should be able to install the resource into an anonymous class created by the test method to avoid this problem.

Reply:
@blairmcg  
Now the Test use an anonymous subclass of Presenter.